docs/servo_power_monitor/src/test_runner.ts

```diff
@@ -2,26 +2,39 @@ import {OperatePort} from './operate_port';
 import {Ui} from './ui';
 
 export class TestRunner {
-  public isOpened = false;
   private CANCEL_CMD = '\x03\n';
+  private isOpened = false;
   // shell script
   private scripts = `#!/bin/bash -e
 function workload () {
-  stress-ng -c 1 -t $1
+  stress-ng -c 1 -t 10
 }
-ectool chargecontrol idle
+ectool chargecontrol idle 1> ./test_out.log 2> ./test_err.log
```

Comment:
How about wrapping those top-level commands that their outputs are redirected into the same file as a function? In that way, we can avoid repeating redirects. e.g.
```
function run_iteration () {
  ectool chargecontrol idle
  workload
  ...
}
run_iteration
```
FYI: there is a Shell style guide: https://google.github.io/styleguide/shellguide.html#s7-naming-conventions

Also, it may be a good idea to refactor the ectool chargecontrol calls and/or put comments on them since it's not clear what they are doing for the people who don't know about ectool well...

Reply:
I agree that repeating redirects is redundant, but I think ectool chargecontrol should be called only once during one measurement. So how about executing the command outside of the script, such as before the cat command?

Reply:
Yeah that seems to be a better solution! Could you update the code to do so?

Reply:
already updated! please check L33, L37 in `test_runner.ts`

Reply:
thanks!! checked!